### PR TITLE
added tutorials

### PR DESCRIPTION
**Pull request by sushanth7781**
This pull request introduces a new Tutorials section to the FreeCAD README file and adds initial tutorial content. The updates aim to enhance the documentation and provide a structured approach for users to get started with FreeCAD. Below are the key changes:
Updated README.md:
Added a new Tutorials section with a link to the tutorial directory.
Improved formatting and fixed minor issues for better readability.
New Tutorial Files:
Created a tutorials/ directory with the following initial content:
index.md: An overview of available tutorials, organized by difficulty level.
Example tutorials for different skill levels, including beginner, intermediate, and advanced topics.
Details:
Added:
tutorials/index.md - Provides an overview and links to individual tutorials.
Placeholder files for beginner, intermediate, and advanced tutorials.
Updated:
README.md - Includes a new section that directs users to the Tutorials for further learning.
Purpose:
The purpose of these changes is to provide new users with a structured way to learn FreeCAD and to make it easier for them to find relevant tutorials. This should improve user onboarding and overall experience with FreeCAD.
Related Issues:
This PR addresses the need for better documentation and user guides, as discussed in [issue #XXXX](https://github.com/FreeCAD/FreeCAD/issues/XXXX) (if applicable).
Notes:
Please review the new tutorial content for accuracy and completeness.
Feedback on additional topics or improvements is welcome.

### CODE REVIEW

File: src/tutorials/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+    # FreeCAD Tutorials
```
Comment (luzpaz) [inline src/tutorials/index.md:1]: Is the whitespace here necessary ?

File: README.md [hunk 2]
```diff
@@ -10,6 +10,132 @@
...
+<!-- <a href="https://freecad.org"><img src="/src/Gui/Icons/freecad.svg" height="100px" width="100px"></a>
+
+### Your own 3D parametric modeler
+
+[Website](https://www.freecad.org) • 
+[Documentation](https://wiki.freecad.org) •
+[Forum](https://forum.freecad.org/) •
+[Bug tracker](https://github.com/FreeCAD/FreeCAD/issues) •
+[Git repository](https://github.com/FreeCAD/FreeCAD) •
+[Blog](https://blog.freecad.org)
```
Comment (luzpaz) [inline README.md:136]: Why are their asterisks at the end of the line for all of these ? Why is this section commented out ?

File: README.md [hunk 3]
```diff
@@ -10,6 +10,132 @@
...
+- [Development](https://wiki.freecad.org/Developer_hub)
+
+The [FreeCAD forum](https://forum.freecad.org) is also a great place
+to find help and solve specific problems you might encounter when
+learning to use FreeCAD.
+
+Tutorials
+---------
+
+Check out our [tutorials](tutorials/index.md) to get started with FreeCAD. The tutorials cover a range of topics from beginner to advanced levels, helping you to master FreeCAD.
```
Comment (luzpaz) [inline README.md:119]: Should be `[tutorials](src/tutorials/index.md)`

**Comment by luzpaz** [comment]
Reviewers: See live example at https://github.com/sushanth7781/FreeCAD/blob/main/src/tutorials/index.md

File: README.md (hunk 2 above)
Comment (sushanth7781) [reply README.md:136]: that is the old readme file which I've just commented out

**Comment by spanner888** [comment]
Is there any additional background or discussion on this request, for example forum discussion or in another FC github issue?
I am hoping that such information might clarify answers to questions below.

In particular is there any discussion on how this will fit with the existing [[Getting started](https://wiki.freecad.org/Getting_started)]?
I ask in terms of both the current and planned future content.

I applaud the very focused simplicity of the Tutorials in the pull request which are nicely task learning based and not just blindly click this & that as is all to common.

I guess that these maybe just a starting outline of what has been intended, but I am unsure of this as there seems to be missing information and steps which leads me to  wonder if these have been reviewed by more experienced users, or some of the key Forum support users and also by any true beginners. 

One example issue I see in the attached tutorials is multiple steps in one step of the tutorial, examples below. This will confuse users. The first example is two steps, but includes three different concepts, so is likely to be more confusing.

[3 Create a new body and start a new sketch on the XY plane.](https://github.com/sushanth7781/FreeCAD/blob/main/src/tutorials/beginner/introduction.md)

[3 Draw a rectangle and close the sketch.](https://github.com/sushanth7781/FreeCAD/blob/main/src/tutorials/beginner/introduction.md)

One common issue that a lot of new users have is actually joining lines in the sketcher. Request for help that are caused by lines users believe are joined appear frequently. So tutorials steps asking users to join lines or create shapes or "close the sketch" should all provide more detail or links to how to join line segments and also how to actually tell if they are joined or how to find segments that are not joined.

Two example tutorial steps that will trip up many users with this learning step are below:

[3 Draw a rectangle and close the sketch.](https://github.com/sushanth7781/FreeCAD/blob/main/src/tutorials/beginner/introduction.md)

[Close the sketch and use the Pad tool to create a 3D object.](https://github.com/sushanth7781/FreeCAD/blob/main/src/tutorials/beginner/basic_sketch.md)

**Comment by Roy-043** [comment]
There has been a forum discussion on how and when we would switch from the Wiki to GitHub for our documentation. This PR completely ignores and undermines that. Currently all documentation should still happen on the FreeCAD Wiki. With our very limited resources we cannot afford to create a 2nd parallel set of documents on GitHub. Please get serious everybody!

Sorry, but IMO this should be closed.

**Comment by yorikvanhavre** [comment]
I agree with @Roy-043 that having tutorials directly inside the FreeCAD source code is not optimal. They are hard to discover and will be mostly invisible for new and inexperienced users, who are targeted by these tutorials.

This kind of very simple beginner-friendly tutorial is however a very good idea. @sushanth7781 would you be ready to add those to https://wiki.freecad.org/Tutorials instead?

**Comment by yorikvanhavre** [comment]
@sushanth7781 could you create yourself a FreeCAD forum account and then give me your forum username here? I'll activate a wiki account for you.

**Comment by chennes** [comment]
Based on this discussion, I'm going to close this PR, since this isn't the best place for us to put tutorials. Please feel free to put this content elsewhere, e.g. the website and/or wiki.

**Pull request closed without merging**